src/main/java/entity/CorporatePage.java

```diff
@@ -1,17 +1,31 @@
 package entity;
 
-public class CorporatePage {
+/**
+ * CorporatePage is a child class of the abstract class Page, which displays company info and internship reviews
+ * Only the CorporateRep user has access to create a CorporatePage instance
+ */
+
+public class CorporatePage extends Page {
+
+    // This uniqueIdTracker starts from 0, and increments by 1 every time it's assigned to a CorporatePage instance.
+    static int uniqueIdTracker = 0;
```

Comment:
When we store a page to the db, it will be provided with an id so there is no need to set this